### PR TITLE
Adding the support for including symlink files when indexing

### PR DESCRIPTION
**Pull request by Peilin-Yang**

### CODE REVIEW

**Comment by lintool** [comment]
Thanks @Peilin-Yang - how is this tested?

@isoboroff do you want to give this a shot and let us know if it works?

File: src/main/java/io/anserini/collection/DocumentCollection.java [hunk 1]
```diff
@@ -108,6 +108,14 @@ public final Path getCollectionPath() {
       public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
         Path name = file.getFileName();
         boolean shouldAdd = true;
+        if (Files.isSymbolicLink(file)) {
```
Comment (lintool) [inline src/main/java/io/anserini/collection/DocumentCollection.java]: Worth adding a comment to reference #481

**Comment by Peilin-Yang** [comment]
@isoboroff 
Could you please try 
```
git fetch
git branch --track issue-481 origin/issue-481
git checkout issue-481
mvn clean package appassembler:assemble
```
and let us know?

**Comment by Peilin-Yang** [comment]
@lintool actually there are somethings:

1. I haven't added the comments that you had suggested
2. I did not consider/test much about what if the symlink points to some files/dirs inside the root path

**Pull request merged**